test/data_map_test.exs

```diff
@@ -0,0 +1,162 @@
+defmodule DataMapTest do
```

Comment:
It's a good point, but it would do a lot of tests... (which would be a good thing..)